validation/src/test/scala/hmda/validation/rules/ts/validity/TsEditCheckSpec.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.ts
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.parser.fi.ts.TsGenerators
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+abstract class TsEditCheckSpec extends PropSpec with PropertyChecks with MustMatchers with TsGenerators {
+  implicit val generatorDriverConfig =
+    PropertyCheckConfig(minSuccessful = 100, maxDiscarded = 500)
+
+  def check: EditCheck[TransmittalSheet]
+
+  // TODO consider trying Matcher instead of these methods
+  implicit class LarChecker(ts: TransmittalSheet) {
```

Comment:
I think this should be named `TsChecker`. There is a [PR](https://github.com/cfpb/hmda-platform/pull/346/files) coming later that is implementing this as well, might be worth taking a look and making sure both implementations are consistent (including the TODO comment)